src/styles/card.scss

```diff
@@ -0,0 +1,82 @@
+
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  flex-wrap: wrap;
```

Comment:
Here you don`t need it

Reply:
when I tried to move it to the body it didn't work. so I had to bring it back